docs/UPGRADE.md

```diff
@@ -38,5 +38,9 @@ uv run pytest --pre-upgrade --post-upgrade --upgrade-deployment-modes=servelerss
 ```
 
 ```bash
-uv run pytest --pre-upgrade --post-upgrade --upgrade-deployment-modes=servelerss,rawdeployment
+uv run pytest --pre-upgrade --post-upgrade --upgrade-deployment-modes=servelerss,rawdeployment,authorino
```

Comment:
Could you please add an explanation of what this command does?
I think it would be helpful to understand what the command is actually doing.